### ISSUE TITLE
compute_space_width: Is 'if "/W" in ft' unreachable?

### ISSUE THREAD

**Issue by MartinThoma** [py-pdf/PyPDF2#1003]
I've parsed all pages of over 22,000 documents. I never hit the following block starting with `if "/W" in ft`:

![image](https://user-images.githubusercontent.com/1658117/174232218-78690db6-6417-4352-bdb0-66239e90cf2f.png)

Could that block be unreachable for some reason?

Looking at "TABLE 5.14 Entries in a CIDFont dictionary", the code seems fine. So it might also just be a very rarely used feature of PDF.

**Comment by pubpub-zz** [comment]
Here is a file : https://github.com/mozilla/pdf.js/raw/master/test/pdfs/bug921409.pdf

**Comment by MartinThoma** [comment]
Thank you!

I've added the following test locally, but still the mentioned section is not reached:

```python
def test_compute_space_width_w():  # L1266, L1267
    url = "https://github.com/mozilla/pdf.js/raw/master/test/pdfs/bug921409.pdf"
    name = "mozilla-bug921409.pdf"
    reader = PdfReader(BytesIO(get_pdf_from_url(url, name=name)))
    for page in reader.pages:
        page.extract_text()
```

**Comment by pubpub-zz** [comment]
met with CIDFontType0/2
extraction has been fixed/improved by PR #1084
